bcda/auth/backend.go

```diff
@@ -1,17 +1,28 @@
 package auth
 
 import (
+	"crypto/rand"
 	"crypto/rsa"
-	"crypto/sha256"
+	"crypto/sha512"
+	"encoding/base64"
+	"errors"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/dgrijalva/jwt-go"
 	log "github.com/sirupsen/logrus"
+	"golang.org/x/crypto/pbkdf2"
 
 	"github.com/CMSgov/bcda-app/bcda/utils"
 )
 
+// The time for hash comparison should be about 1s.  Increase hashIter if this is significantly faster in production.
```

Comment:
How are we going to know anything about the performance characteristics of the hash at runtime? I didn't see any logging of runtime performance measurements, such that we could set up a splunk alert if the measurement was outside of an acceptable range.

Reply:
Good thought.  The speed check has been moved from a unit test to a `warning` level log which we can alert on in production environments.